### ISSUE TITLE
Can't encode unicode symbol /u039f (greek letter omicron)

### ISSUE THREAD

**Issue by tobischiebel** [thepanacealab/SMMT#21]
```
currently getting 1808100 - 1808200
creating ziped master json file
creating minimized json master file
creating CSV version of minimized json master file
Traceback (most recent call last):
  File "get_metadata.py", line 187, in <module>
    main()
  File "get_metadata.py", line 183, in main
    f.writerow([data["favorite_count"], data["source"], data["text"].encode('utf-8'), data["in_reply_to_screen_name"], data["is_retweet"], data["created_at"], data["retweet_count"], data["id_str"].encode('utf-8')])
  File "C:\Users\USERNAME\AppData\Local\Programs\Python\Python37-32\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u039f' in position 16: character maps to <undefined>
```

**Comment by dream1438** [comment]
did u get any solution?

**Comment by dream1438** [comment]
replace line 

f.writerow([data["favorite_count"], data["source"], data["text"].encode('utf-8'), data["in_reply_to_screen_name"], data["is_retweet"], data["created_at"], data["retweet_count"], data["id_str"].encode('utf-8')])
-----------------------------------
f.writerow([data["favorite_count"], data["source"].encode('utf-8'), data["text"].encode('utf-8'), data["in_reply_to_screen_name"], data["is_retweet"], data["created_at"], data["retweet_count"], data["id_str"].encode('utf-8')])

------------------------------------------------------------------------
in other words, change the  data["source"]  to data["source"].encode('utf-8')
